content/summary-resource/index.md

```diff
@@ -0,0 +1,105 @@
+---
+rfc: 0014
+start_date: 2018-08-13
+pr:
+status: draft
+---
+
+# Summary resource
+
+## Summary
+
+This RFC proposes a summary resource to offer an overview of the current state
+of the register.
+
+Dependencies:
+
+* [RFC0013: Multihash](https://github.com/openregister/registers-rfcs/pull/26)
+
+
+## Motivation
+
+The current summary is exposed as the register resource (`GET /register`).
+The name of the resource is misleading as you don't get the register, only
+some metadata about it. Also, the current specification and implementation
+don't agree on what is the register resource.
+
+## Explanation
+
+The summary resource is a new resource that will replace the old register
+resource. The summary resource will not contain any information about fields
+as this will be addressed by the schema resource.
+
+The most noticeable change is the addition of the `hashing-algorithm` object
+describing the hashing algorithm used in the register to create hashes for
+items, entries and proofs.
+
+***
+### Endpoint
+
+```
+GET /summary
+```
+
+### Response attributes
+
+|Name|Type|Description|
+|-|-|-|
+|`copyright`| String |The copyright of the data.|
+|`custodian`| Optional String |The custodian of the register.|
+|`description`| Optional String |The description of the register.|
+|`hashing-algorithm`| [HashingAlgorithm](#hahing-algorithm-attributes) |The hahing algorithm used throught the register.|
+|`root-hash`| Hash | The Merkle tree root hash. |
+|`last-updated`| Timestamp |The date the register was last updated.|
+|`licence`| String |The licence of the data.|
```

Comment:
What is a valid string for this? Should we require an open source licence?

Reply:
I think we can't afford to be restrictive here. OGL is one of many and these things change too often to be prescriptive at the spec level.

Reply:
Just wondering if there's a way we can encourage some canonical naming so we don't end up with a bunch of registers using different variations of the same licence name.

Reply:
We could add a non-normative section in the spec suggesting something on the lines of https://help.github.com/articles/licensing-a-repository/#searching-github-by-license-type

Or, we could consider having a register for known licenses. I'm not a 100% on board with this one because the custodianship model applied to this but it would be a very useful register to have. What do you think?

Reply:
Something like that list would help I think.

I don't know if it should be register, but maybe the open standards team could comment on that. I think it would be useful if someone ever builds a catalog of reusable software/data across government, like [code.gov](https://code.gov)

Reply:
This seems like a good list to base our restrictions https://spdx.org/licenses/

Reply:
Mentioned in here https://github.com/alphagov/open-standards/issues/31

Reply:
Cool - I think a non-non normative recommendation to use their identifiers would address my concern above.